pyartcd/pyartcd/pipelines/ocp4_konflux.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import os
+from typing import Optional
+
+import click
+from artcommonlib import exectools
+
+from pyartcd import constants, util
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+LOGGER = logging.getLogger(__name__)
+
+
+class KonfluxOcp4Pipeline:
+    def __init__(self, runtime: Runtime, assembly: str, data_path: Optional[str],
+                 image_list: Optional[str], version: str, data_gitref: Optional[str],
+                 kubeconfig: Optional[str]):
+        self.runtime = runtime
+        self.image_list = image_list
+        self._doozer_working = os.path.abspath(f'{self.runtime.working_dir / "doozer_working"}')
+        self.version = version
+        self.kubeconfig = kubeconfig
+
+        group_param = f'--group=openshift-{version}'
+        if data_gitref:
+            group_param += f'@{data_gitref}'
+
+        self._doozer_base_command = [
+            'doozer',
+            f'--assembly={assembly}',
+            f'--working-dir={self._doozer_working}',
+            f'--data-path={data_path}',
+            group_param
+        ]
+
+    async def rebase(self, version: str, input_release: str):
+        cmd = self._doozer_base_command.copy()
+        image_list = self.image_list or ''
+        cmd.extend([
+            '--latest-parent-version',
+            f'--images={image_list}',
+            'beta:images:konflux:rebase',
+            f'--version=v{version}',
+            f'--release={input_release}',
+            f"--message='Updating Dockerfile version and release {version}-{input_release}'",
+        ])
+        if not self.runtime.dry_run:
+            cmd.append('--push')
+        await exectools.cmd_assert_async(cmd)
+
+    async def build(self):
+        cmd = self._doozer_base_command.copy()
+        image_list = self.image_list or ''
+        cmd.extend([
+            '--latest-parent-version',
+            f'--images={image_list}',
+            'beta:images:konflux:build',
```

Comment:
we'd also need `--konflux-namespace ocp-art-tenant` for p02 cluster?

Reply:
Yes, let me add it.